### PR TITLE
Fix Broken Link for Docker Registry Protocol

### PR DESCRIPTION
**Pull request by danielhelfand**
### Description of Changes

Fixing a broken link in the README for documentation on the Docker Registry Protocol. Also adding some ands!

#### Changes:

Added the following link to replace broken link:

https://docs.docker.com/registry/spec/api/

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,18 @@ Project Quay builds, stores, and distributes your container images.
 
 High-level features include:
 
-- Docker Registry Protocol [v1], [v2]
+- Docker Registry Protocol [v1 and v2]
```
Comment (josephschorr) [inline README.md]: Since there apparently isn't a V1 page on Docker's site anymore, can you remove linking v1 to it?

Comment (danielhelfand) [reply README.md]: Sure thing. Just pushed something up.
